skrifa/src/variation.rs

```diff
@@ -168,9 +168,8 @@ impl<'a> AxisCollection<'a> {
     /// * Omitted settings are set to 0.0, representing the default position
     ///     in variation space.
```

Comment:
I suggest you might explicitly specify set to the _normalized_ value 0.0, which is equivalent to the user value at the default in fvar; if you start from user coordinates and read to here I think it would be easy to miss that you said they'd be normalized. Also precious few people grok font coordinates and as much as possible we'd like this all to be somewhat noob accessible.

Reply:
Also, optional, perhaps change "Omitted settings" to "If no setting for an axis is provided" to match teh language used for the prior bullet?

Reply:
Thanks, I like this working better. Changed to:
```
If no setting for an axis is provided, the associated coordinate is
set to the normalized value 0.0, representing the default position
in variation space.
```